pyproject.toml

```diff
@@ -286,6 +288,24 @@ lint.ignore = [
     "E402", # module-level import not at top of file
 ]
 
+[tool.mypy]
+python_version = "3.12"
```

Comment:
I think `python_version` should be omitted here. Leaving it undefined defaults to the Python interpreter that is currently building the program.

https://mypy.readthedocs.io/en/stable/config_file.html#confval-python_version

Reply:
done